src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/configuration/ConfigurationSpec.groovy

```diff
@@ -20,6 +22,10 @@ This spec assumes that 'transit_vlan' is always default type
 """)
 @Isolated //kilda config updates
 class ConfigurationSpec extends HealthCheckSpecification {
+    @Autowired
+    @Shared
+    KildaConfiguration kildaConf
```

Comment:
1. Let's move this to `BaseSpecification`
2. I'd stay with `kildaConfiguration` variable name.